presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeWindowQueries.java

```diff
@@ -189,4 +201,34 @@ public void testLag()
         testWindowFunction("lag(orderkey, 5)", FunctionType.VALUE);
         testWindowFunction("lag(totalprice, 2, -123.456)", FunctionType.VALUE);
     }
+
+    @Test
```

Comment:
Nit : There isn't a need for separate test functions. They can be combined in a single test function.

Reply:
Thanks for the suggestion, updated.